models/dave_robot_models/description/rexrov/model.sdf

```diff
@@ -584,5 +584,22 @@
       <joint_name>thruster8_joint</joint_name>
     </plugin>
 
+    <!-- Sea Pressure Sensor Plugin -->
+    <plugin
+      filename="sea_pressure_sensor"
+      name=" dave_gz_sensor_plugins::SubseaPressureSensorPlugin">
+      <namespace>rexrov</namespace>
+      <!-- <link_name>base_link</link_name> -->
+      <topic>sea_pressure</topic>
+      <!-- <mass>0.015</mass> -->
+      <update_rate>10</update_rate>
+      <!-- <saturation>3000</saturation> -->
+      <noise_sigma>3.0</noise_sigma>
+      <noise_amplitude>0.0</noise_amplitude>
+      <estimate_depth_on>false</estimate_depth_on>
```

Comment:
I've treid `true` and nothing is added to the published msg. I suppose this was to add estimated depth value to the msg but it did not. please check.

Reply:
Fixed this issue in the upcoming commits : )